### PR TITLE
#129: Should work with owners different from "buildo" (closes #129)

### LINKED ISSUE #129: Should work with owners different from "buildo"
## requirements

We want @nemobot to work on repos of any owner
## specs

The owner `"buildo"` is hardcoded in our code so currently we can't use nemobot outside buildo.

We should try to dynamically get the owner from gh webhooks, info.
We should also add that info to synthetic events (like reminders).
## misc

{optional: other useful info}

### PR DESCRIPTION
**Pull request by FrancescoCioria**
Issue #129
### tests performed
- webhook event from this repo -> `githubConfig`
  - `{ name: 'nemobot', org: 'buildo', isEnterprise: false, rootURL: 'https://api.github.com', token: ..., github: ...`}
- webhook event from `aliniq` -> `githubConfig`
  - `{ name: 'aliniq', org: 'buildo', isEnterprise: true, rootURL: 'https://github.omnilab.our.buildo.io/api/v3', token: ..., github: ...`}
- webhook event from test personal repo  -> `githubConfig`
  - `{ name: 'nemobot-tests', org: 'FrancescoCioria', isEnterprise: false, rootURL: 'https://api.github.com', token: ..., github: ...`}
- add webhook on `FrancescoCioria/nemobot-tests`
  - labels normalized (`normalization.js`)
- create issue
  - after 10 seconds reminds to add topic labels! (`reminder.js`)
- create sub-issue
  - added label `macro`
  - added `sub-issue` chapter
- close sub-issue
  - macro issue is closed!
- re-open sub-issue
  - macro issue is re-opened!
### tests not performed (domain coverage)

@nemobot handles tons of scenarios... I tested the most common ones

### CODE REVIEW

**Comment by FrancescoCioria** [comment]
@gabro we currently have three custom events: "split-macro-issue", "branch-preview" and `hophop` that adds the label "WIP"

- the first one is still in beta as the UI on worflow-pal is still to be implemented.
- "branch-preview" can't be used with browserstack -> at the moment is legacy code

The only one that causes a breaking change is the `hophop` event. I already addressed it with this PR on `hophop`: https://github.com/buildo/hophop/pull/69

before merging we should publish a new version for `hophop`

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "clean": "rm -rf lib && mkdir lib",
     "build": "npm run clean && babel src --out-dir lib",
     "watch": "npm run clean && babel src --out-dir lib --watch",
-    "start": "DEBUG=prettifier*,http,express:* node lib/index.js"
+    "start": "DEBUG=prettifier*,nemobot*,http,express:* node lib/index.js"
```
Comment (gabro) [inline package.json]: yeah, actually `prettifier` could be deprecated in favor of `nemobot`

**Comment by gabro** [comment]
If you already published a version of `hophop` I'm 👍 for merging

**Comment by FrancescoCioria** [comment]
> If you already published a version of hophop I'm 👍 for merging

nope, please wait. I'll do both in the following days :)

File: package.json (hunk 1 above)
Comment (FrancescoCioria) [inline package.json]: yes I know, we should open an issue

**Pull request merged**